application/asana/v0/config/setup.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "properties": {
+    "token": {
+      "description": "Fill in your Asana Personal Access Token (PAT). You can generate one from <a href=\"https://app.asana.com/0/my-apps\">developer console<a/>",
```

Comment:
```suggestion
  "additionalProperties": false,
  "properties": {
    "token": {
      "description": "Fill in your Asana Personal Access Token (PAT). You can generate one from [developer console](https://app.asana.com/0/my-apps)",
```

Reply:
I'd like to know why I should change the link to md format. It was not working in the previous console UI or the documentation. Is it fixed now?

Reply:
Originally, the IDE shows error
And, now, I found the reason..
<a/> should be </a>
![image](https://github.com/user-attachments/assets/8b71789e-5c71-4ea1-a708-42f5b4d14f3d)